layers/newrelic/README.md

```diff
@@ -0,0 +1,85 @@
+# Bref New Relic Extension
```

Comment:
Let's put this file in the /doc directory

Reply:
No problem. Moving now and resubmitting.